test/verify/check-openshift

```diff
@@ -787,7 +787,7 @@ class TestRegistry(MachineCase):
         b.set_val("#user_name", "bar ^ baz")
         b.set_val("#identities", "anypassword:abc123")
         b.click(".btn-primary")
-        b.wait_in_text(".dialog-error", "The member name contains invalid characters.")
+        b.wait_in_text(".dialog-error", "The name contains invalid characters.")
```

Comment:
That error string is generated [here](https://github.com/cockpit-project/cockpit/blob/master/pkg/kubernetes/scripts/projects.js#L924), so it seems the change above short-circuits the extended RE for member names in this case. The test below verifies that email-style member names are accepted though, and thus should catch regressions. Thus, retrying the failed tests, I would like them to be green before landing this.